pkg/sync/task.go

```diff
@@ -72,18 +76,85 @@ func (t *Task) Run() error {
 			// print the log of ignored blob
 			t.Infof("Blob %s(%v) has been pushed to %s, will not be pulled", b.Digest, b.Size, t.destination.GetRegistry()+"/"+t.destination.GetRepository())
 		}
+
 	}
 
-	// push manifest to destination
-	if err := t.destination.PushManifest(manifestByte); err != nil {
-		return t.Errorf("Put manifest to %s/%s:%s error: %v", t.destination.GetRegistry(), t.destination.GetRepository(), t.destination.GetTag(), err)
+	//Push manifest list
+	if manifestType == manifest.DockerV2ListMediaType {
+		manifestSchemaListInfo, err := manifest.Schema2ListFromManifest(manifestByte)
+		if err != nil {
+			return err
+		}
+
+		var subManifestByte []byte
+
+		// push manifest to destination
+		for _, manifestDescriptorElem := range manifestSchemaListInfo.Manifests {
+
+			if manifestDescriptorElem.Platform.Architecture == "amd64" || manifestDescriptorElem.Platform.Architecture == "arm64" {
```

Comment:
Will we get into trouble if not only for "amd64" and "arm64" ? I'm wondering if other architectures can also be supported here.

Reply:
no trouble to support other arch, just for save storage space

Reply:
image-syncer use system memory to sync images, maybe it's not necessary.

Reply:
i have removed the limit